workers/dockerfiles/Dockerfile.salmon

```diff
@@ -69,7 +64,7 @@ COPY common/dist/data-refinery-common-* common/
 RUN pip3 install common/$(ls common -1 | sort --version-sort | tail -1)
 
 USER user
-
+RUN echo "export PATH=`pwd`/Salmon-latest_linux_x86_64/bin:$PATH" >> ~/.bashrc
```

Comment:
I didn't put it in `~/.profile` or `~/.bash_profile` because we would like the new PATH to be effective in all bash sessions (not only in BASH login shell).